frontend/components/UI/NavBar.js

```diff
@@ -41,21 +41,6 @@ export default function NavBar(props) {
                 },
             } = authData;
             props.handleLensLogin(accessToken, refreshToken);
-
-            // TODO - check for user existing in our contract using graph call
-            // before going on and creating the user 
-            const { data : profileData } = await lensClient.query({
-                query: getProfileByAddress,
-                variables: {
-                    owner: lensaddress,
-                },
-            });
-
-            const lensHandle = profileData?.profiles?.items[0]?.handle;
-
-            if (lensHandle) {
-                await createIPFontsUser({ lensAddress: lensaddress, lensHandle });
-            }
```

Comment:
Will this logic be moved to the same place where the user metadata upload happens?

Reply:
Yeah exactly so we would call `createIPFontsUser` when a user updates their lens profile for the first time through our dapp if they have a lens handle or the user creates a new profile when the they don't have a lens handle.